lib/jsonapi_parameters/translator.rb

```diff
@@ -1,16 +1,20 @@
 module JsonApi::Parameters
-  def jsonapify(params)
-    jsonapi_translate(params)
+  def jsonapify(params, naming_convention: :snake)
```

Comment:
There is something like that written in the docs:
> If the input is in a different convention than `:snake_case` (...)

So as far as I see there is a small difference between the docs and the implementation

Reply:
Good catch, you are correct!